pkg/doc/jsonld/embed_contexts.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jsonld
+
+import "embed"
+
+// EmbedFS contains JSON-LD context documents from the "embed_contexts" folder.
+//go:embed embed_contexts/*.jsonld
+var EmbedFS embed.FS //nolint:gochecknoglobals
+
+// EmbedContexts contains a list of context documents embedded into Go binary.
+var EmbedContexts = []ContextDocument{ //nolint:gochecknoglobals
+	{
+		URL:         "https://www.w3.org/2018/credentials/v1",
+		DocumentURL: "",
+		Path:        "embed_contexts/credentials_v1.jsonld",
+	},
+	{
+		URL:         "https://www.w3.org/2018/credentials/examples/v1",
+		DocumentURL: "",
+		Path:        "embed_contexts/credentials-examples_v1.jsonld",
+	},
+	{
+		URL:         "https://w3id.org/security/v1",
+		DocumentURL: "https://w3c-ccg.github.io/security-vocab/contexts/security-v1.jsonld",
+		Path:        "embed_contexts/security_v1.jsonld",
+	},
+	{
+		URL:         "https://w3id.org/security/v2",
+		DocumentURL: "https://w3c-ccg.github.io/security-vocab/contexts/security-v2.jsonld",
+		Path:        "embed_contexts/security_v2.jsonld",
+	},
+	{
+		URL:         "https://w3id.org/security/jws/v1",
+		DocumentURL: "https://w3c-ccg.github.io/lds-jws2020/contexts/v1/",
+		Path:        "embed_contexts/lds-jws2020_v1.jsonld",
+	},
+	{
+		URL:         "https://w3id.org/security/bbs/v1",
+		DocumentURL: "https://w3c-ccg.github.io/ldp-bbs2020/contexts/v1/",
+		Path:        "embed_contexts/ldp-bbs2020_v1.jsonld",
+	},
+	{
+		URL:         "https://w3id.org/did/v0.11",
+		DocumentURL: "https://w3c-ccg.github.io/did-spec/contexts/did-v0.11.jsonld",
+		Path:        "embed_contexts/did_v0.11.jsonld",
+	},
+	{
+		URL:         "https://w3id.org/did/v1",
+		DocumentURL: "https://w3c-ccg.github.io/did-spec/contexts/did-v1.jsonld",
+		Path:        "embed_contexts/did_v1.jsonld",
+	},
+	{
+		URL:         "https://www.w3.org/ns/did/v1",
+		DocumentURL: "https://www.w3.org/ns/did/v1",
+		Path:        "embed_contexts/did_v1.jsonld",
+	},
+	{
+		URL:         "https://w3id.org/citizenship/v1",
```

Comment:
We shouldn't be embedding specific vc vocabs or examples at the framework level. You should allow for the app or SDK built on top of the framework to add their own embeds. I would imagine you would have a common set of embeds for framework tests.

Reply:
ok, I am considering moving them to [jsoldtest](https://github.com/hyperledger/aries-framework-go/tree/083926d9b9233f460c061eeb59fb3c331093c083/pkg/internal/jsonldtest) package